model/src/main/java/org/opennms/poc/ignite/model/workflows/Workflow.java

```diff
@@ -15,6 +17,7 @@ public class Workflow implements Serializable {
     private Map<String,String> parameters = new LinkedHashMap<>();
     private String uuid;
     private String cron;
+    private List<FieldConfigMeta> fieldConfigMeta;
```

Comment:
As suggested before - I don't think we need to send this back to worker. Worker should have access to `FieldConfigMetadata` based on its own runtime state.

Reply:
Right, we have taken this as something that needs to be resolved later.